Issue4〜6の修正

poponosan opened the pull request:
Issue4〜6の解消

poponosan committed:
Issue4

poponosan committed:
Issue5

poponosan committed:
issue6

poponosan committed:
issue4~6のプルリクエスト修正の為のコミット

Gyazoの動画を貼り付けるのを忘れていました。

poponosan commented:
Gyazoの動画を貼りつけるのを忘れていました。

https://gyazo.com/bb04f5c23929a7662cfdd3992d8daf32

maimai0401 reviewed (commented):
レビュー依頼ありがとうございます。
コメントいたしましたので、ご確認をお願いいたします。
修正後、再度レビュー依頼フォームにてご提出くださいませ。

poponosan committed:
前回のissueの修正内容

前回のissueの修正内容が元に戻っていたので修正しました。

『Issue１〜３』
①不要なdivタグの削除
②シンボル型へ修正
③def getWeekの修正

poponosan commented:
修正しましたのでご査収のほどお願いいたします。

maimai0401 reviewed (commented):
レビュー依頼ありがとうございます。
修正点確認させていただきました。
コメントいたしましたので、ご確認お願いいたします。
修正後、再度レビュー依頼フォームにてご提出くださいませ。

poponosan committed:
issueの1~3の修正

get_Weekが大文字になっていたので小文字に修正

poponosan commented:
get_Weekの「w」が大文字になっていたので小文字に修正しました。
